### PR TITLE
fix waitOnSearchComplete loop issue

### PR DESCRIPTION
**Pull request by yen-tt**
- update hook registration to use `requestHook` on fixture instead of `beforeEach` for each test
- update waitOnSearchComplete to use current testInstance from test suite instead of the imported testInstance reference from testcafe
- update tests on searchThisArea functionality and drag on search

### CODE REVIEW

**Comment by oshi97** [comment]
is there something we can do about the search when map moves test? it's still marked as unstable

File: tests/acceptance/suites/vertical-full-page-map.js [hunk 1]
```diff
@@ -30,48 +33,44 @@ test('Clicking on a pin focuses on a results card', async t => {
...
 test('Search when map moves works', async t => {
   await SearchBar.submitQuery('virginia');
-  await SearchRequestLogger.waitOnSearchComplete();
-  const resultsCountBeforeDrag = await VerticalResults.getNumResults();
+  await SearchRequestLogger.waitOnSearchComplete(t);
   await ThemeMap.dragLeft();
   await ThemeMap.dragLeft();
-  await SearchRequestLogger.waitOnSearchComplete();
-  const resultsCountAfterDrag = await VerticalResults.getNumResults();
-  await t.expect(resultsCountBeforeDrag !== resultsCountAfterDrag).ok();
```
Comment (oshi97) [inline tests/acceptance/suites/vertical-full-page-map.js:39]: do we no longer want this expect?

File: tests/acceptance/suites/vertical-full-page-map.js [hunk 2]
```diff
@@ -4,23 +4,26 @@ import VerticalResults from '../blocks/verticalresults';
 import ThemeMap from '../blocks/thememap';
 import Pagination from '../blocks/pagination';
 import SearchRequestLogger from '../searchrequestlogger';
+import { VERTICAL_SEARCH_URL_REGEX } from '../constants';
+import { registerIE11NoCacheHook } from '../../test-utils/testcafe';
 
 fixture`Vertical Full Page Map`
   .page(`http://localhost:${PORT}/locations_full_page_map`)
+  .requestHooks(SearchRequestLogger.createVerticalSearchLogger())
```
Comment (oshi97) [inline tests/acceptance/suites/vertical-full-page-map.js:12]: like this!

File: tests/acceptance/suites/vertical-full-page-map.js (hunk 1 above)
Comment (yen-tt) [reply tests/acceptance/suites/vertical-full-page-map.js:39]: the goal of this test is to see that dragging the map would trigger a search. Previously, we saved the results before and after to compare and check for a search. Now, we can just use the requestLogger to confirm that a search is fired!

**Review by oshi97 — APPROVED** [review]

**Pull request merged**